handbook.md

```diff
@@ -724,6 +724,7 @@ The goal of Atlantian Youth Rapier is to recreate pre-17th century fencing.  All
         #. Rapier (two-handed sword)
         #. Cut & Thrust
         #. Cut & Thrust (two-handed sword)
+        #. Rapier Spear
```

Comment:
Develop objective skills and safety verification.  Add training for Ushers.

Reply:
I agree that defining authorization expectations is important, these are currently defined in the [Rapier Marshal 101 handbook](https://marshal.atlantia.sca.org/training/rapier-marshal/).  As an example, C&T expectations are defined in that document.  This mirrors the Armored Combat for Atlantia, which defines weapon specific expectations in [Armored Marshal 101 handbook](https://marshal.atlantia.sca.org/training/armored-marshal/#authorizations)